### DISCUSSION TITLE
Help: Compiled .exe file but missing .dll file

### DISCUSSION THREAD

**Discussion by fdewithph** [Anura3D/Anura3D_OpenSource#268] [Q&A]
Hi everyone,
this is my version VS2019 16.11.41,latest oneAPI. After compiling, I only got the .exe file, but the expected .dll file is missing.
I’ve already checked for issues like runtime library conflicts (for example, I encountered warning LNK4098: "default library 'LIBCMTD' conflicts with other libraries; use /NODEFAULTLIB") and missing PDB files (such as warning LNK4099: "PDB 'hdf5.pdb' not found"). but I'm not sure what led to this
Has anyone faced a similar issue before? Any suggestions on how to fix this and ensure the .dll file is created alongside the .exe file?
Thanks in advance for any help!
<img width="952" alt="Q1" src="https://github.com/user-attachments/assets/9508a3ff-2ebf-4fe9-a6f1-e57bab61bddc">
<img width="962" alt="Q2" src="https://github.com/user-attachments/assets/6849aaa4-b530-4cb1-9853-c2371ccb7c4c">

**Comment by dj798** [comment]
你好 请问咋解决的啊
![Uploading 123.png…]()
